### PR TITLE
Update Aerie Dependencies

### LINKED ISSUE #1213: Update Aerie dependiencies
We have some dependencies that need to be updated in the repo. Additionally, we have some dependencies that are unused and should be cleaned up.

### PR DESCRIPTION
**Pull request by Mythicaeda**
* **Tickets addressed:** Closes #1213, Closes #1203 
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description

- Removes dependencies that are unused from our `build.gradle`s
- Replaces `assertJ` and `GoogleTruth` statements with equivalent `JUnit` statements for consistency
- Updates the sequencing sever and load tests `package-lock.json`s
- Updates Javalin to 5.6.3 (prior version was 5.6.2)

These changes are expected to resolve all of our current active security warnings.

## Verification

Tests in the `merlin-server`, `scheduler-driver`, and `scheduler-server` packages were updated to remove `assertJ` or `Google Truth` dependencies.

## Documentation

No docs need to be updated

### CODE REVIEW

File: scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/PrioritySolverTest.java [hunk 1]
```diff
@@ -119,8 +117,16 @@ private static PlanInMemory makePlanAB012(Problem problem) {
...
-  private static final Correspondence<SchedulingActivityDirective, SchedulingActivityDirective> equalExceptInName = Correspondence.from(
-      SchedulingActivityDirective::equalsInProperties, "matches");
+  public static void assertActivitiesListsEquals(List<SchedulingActivityDirective> expected, List<SchedulingActivityDirective> actual) {
+    assertEquals(expected.size(), actual.size());
+
+    final var sortedExpected = expected.stream().sorted(Comparator.comparing(SchedulingActivityDirective::startOffset)).toList();
+    final var sortedActual = actual.stream().sorted(Comparator.comparing(SchedulingActivityDirective::startOffset)).toList();
+
+    for(int i = 0; i < sortedExpected.size(); ++i){
+      assertTrue(sortedExpected.get(0).equalsInProperties(sortedActual.get(0)));
```
Comment (adrienmaillard) [inline scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/PrioritySolverTest.java]: index is always 0

File: scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/PrioritySolverTest.java [hunk 2]
```diff
@@ -119,8 +117,16 @@ private static PlanInMemory makePlanAB012(Problem problem) {
   }
 
   /** matches activities if they agree in everything except the (possibly auto-generated) names **/
-  private static final Correspondence<SchedulingActivityDirective, SchedulingActivityDirective> equalExceptInName = Correspondence.from(
-      SchedulingActivityDirective::equalsInProperties, "matches");
+  public static void assertActivitiesListsEquals(List<SchedulingActivityDirective> expected, List<SchedulingActivityDirective> actual) {
+    assertEquals(expected.size(), actual.size());
+
+    final var sortedExpected = expected.stream().sorted(Comparator.comparing(SchedulingActivityDirective::startOffset)).toList();
+    final var sortedActual = actual.stream().sorted(Comparator.comparing(SchedulingActivityDirective::startOffset)).toList();
```
Comment (adrienmaillard) [inline scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/PrioritySolverTest.java]: we may have a lot of activities starting at the same time making the sort unstable for that purpose

Comment (Mythicaeda) [reply scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/PrioritySolverTest.java]: I've updated the helper method to account for that. It's also been moved into `TestUtility`

**Review by dandelany — APPROVED** [review]
Looks good to me 👍 Changes seem sensible, I tested creating plans and simulating after rebuilding & recreating docker containers locally with no issues.

File: scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/TestUtility.java [hunk 3]
```diff
@@ -146,4 +152,23 @@ public static Instant timeFromEpochDays(int days) {
     return timeFromEpochHours(days * 24);
   }
 
+  /** matches activities if they agree in everything except the (possibly auto-generated) names **/
+  public static void assertActivitiesListsEquals(List<SchedulingActivityDirective> expected, List<SchedulingActivityDirective> actual) {
```
Comment (adrienmaillard) [inline scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/TestUtility.java]: Can we change the name of this procedure by assertSetEquality ?

**Review by adrienmaillard — APPROVED** [review]
LGTM

File: scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/TestUtility.java (hunk 3 above)
Comment (Mythicaeda) [reply scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/TestUtility.java]: Done 👍

**Pull request merged**